File: frontend-api/src/main/java/no/nav/melosys/tjenester/gui/dto/OppgaveOversiktDto.java
package no.nav.melosys.tjenester.gui.dto;

import java.util.List;

import no.nav.melosys.service.oppgave.dto.BehandlingsoppgaveDto;
import no.nav.melosys.service.oppgave.dto.JournalfoeringsoppgaveDto;

public class OppgaveOversiktDto {
    private List<JournalfoeringsoppgaveDto> journalforing;
    private List<BehandlingsoppgaveDto> saksbehandling;

    public List<JournalfoeringsoppgaveDto> getJournalforing() {
        return journalforing;
    }

    public void setJournalforing(List<JournalfoeringsoppgaveDto> journalforing) {
        this.journalforing = journalforing;
    }

    public List<BehandlingsoppgaveDto> getSaksbehandling() {
        return saksbehandling;
    }

    public void setSaksbehandling(List<BehandlingsoppgaveDto> saksbehandling) {
        this.saksbehandling = saksbehandling;
    }
}

File: frontend-api/src/main/kotlin/no/nav/melosys/tjenester/gui/dto/oppgave/OppgaveOversiktDto.kt
package no.nav.melosys.tjenester.gui.dto.oppgave

import no.nav.melosys.service.oppgave.dto.BehandlingsoppgaveDto
import no.nav.melosys.service.oppgave.dto.JournalfoeringsoppgaveDto

data class OppgaveOversiktDto(
    val journalforing: List<JournalfoeringsoppgaveDto>,
    val saksbehandling: List<BehandlingsoppgaveDto>,
)
